### PR TITLE
Squared Donut Shape Added

### LINKED ISSUE #61: Square Frame like Donut!
I can add this Please assign it to me!

### PR DESCRIPTION
**Pull request by Spyware007**
Closes #61 New Shape Added

### CODE REVIEW

**Comment by Spyware007** [comment]
@Sachin-chaurasiya  Please review this and 
when I am trying to add the type as a string in [src/lib/shapes/data/squaredDonutPath.ts](https://github.com/ashutosh1919/react-awesome-shapes/pull/63/files#diff-452cc63d724e9c83eca4d7e3f5c648b9705245b610d1fe1c2d87d1d15a2c997b) it gives me an error while committing the error says "error Type string trivially inferred from a string literal, remove type annotation @typescript-eslint/no-inferrable-types"

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya Please review this and when I am trying to add the type as a string in [src/lib/shapes/data/squaredDonutPath.ts](https://github.com/ashutosh1919/react-awesome-shapes/pull/63/files#diff-452cc63d724e9c83eca4d7e3f5c648b9705245b610d1fe1c2d87d1d15a2c997b) it gives me an error while committing the error says "error Type string trivially inferred from a string literal, remove type annotation @typescript-eslint/no-inferrable-types"

@Spyware007 , That's because typescript inferred the type from the assigned value.

File: src/lib/shapes/data/squaredDonutPath.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const squaredDonutPath: any = `polygon(
```
Comment (Sachin-chaurasiya) [inline src/lib/shapes/data/squaredDonutPath.ts:1]:
```suggestion
export const squaredDonutPath = `polygon(
```

File: src/components/demo/shapesdemos.tsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ import {
   Hexagon,
   Polygon,
   Star,
-  Cross
+  Cross,
+  SquaredDonut
```
Comment (Sachin-chaurasiya) [inline src/components/demo/shapesdemos.tsx:16]:
```suggestion
  SquareDonut
```
Make the required changes in other places as well.

**Comment by Sachin-chaurasiya** [comment]
@Spyware007 , Please do not make changes from github directly. 
Please do the changes from the editor so that, we don't get any linting issues.

**Comment by Sachin-chaurasiya** [comment]
@Spyware007 , there is some linting issue on your PR so please resolved those errors.

**Pull request closed without merging**